evalml/tests/component_tests/test_oversampler.py

```diff
@@ -474,19 +474,8 @@ def test_oversampler_handle_nullable_types(
         im.SMOTEN(),
     ],
 )
-@pytest.mark.parametrize(
-    "handle_incompatibility",
-    [
-        True,
-        pytest.param(
-            False,
-            marks=pytest.mark.xfail(strict=True, raises=ValueError),
-        ),
-    ],
-)
 def test_oversampler_nullable_type_incompatibility(
```

Comment:
How would you feel about removing this test completly? We have the test above this that checks that we can pass in nullable types without erroring (which I guess is passing along with this one because we don't run min dependency tests with it?)

Reply:
ah I missed that - can remove. They're both failing for the min tests: https://github.com/alteryx/evalml/actions/runs/4376580073/jobs/7658889954